Sources/PrimerSDK/Classes/Core/Primer/PrimerDelegate.swift

```diff
@@ -34,6 +34,13 @@ public protocol PrimerDelegate: class {
     func checkoutFailed(with error: Error)
 }
 
+public extension PrimerDelegate {
```

Comment:
lgtm! 

Thing is that this is the first thing I tried, and I remember getting an error. I have no idea why, and I don't remember what it was.